ui/angular.json

```diff
@@ -0,0 +1,179 @@
+{
+  "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
+  "version": 1,
+  "newProjectRoot": "projects",
+  "projects": {
+    "aou-workbench-ui": {
+      "root": "",
+      "sourceRoot": "src",
+      "projectType": "application",
+      "architect": {
+        "build": {
+          "builder": "@angular-devkit/build-angular:browser",
+          "options": {
+            "outputPath": "dist",
+            "index": "src/index.html",
+            "main": "src/main.ts",
+            "tsConfig": "src/tsconfig.app.json",
+            "polyfills": "src/polyfills.ts",
+            "assets": [
+              "src/assets",
+              "src/favicon.ico"
+            ],
+            "styles": [
+              "node_modules/@clr/icons/clr-icons.min.css",
+              "node_modules/@clr/ui/clr-ui.min.css",
+              "node_modules/nouislider/distribute/nouislider.min.css",
+              "src/styles.css"
+            ],
+            "scripts": [
+              "node_modules/mutationobserver-shim/dist/mutationobserver.min.js",
+              "node_modules/@webcomponents/custom-elements/custom-elements.min.js",
+              "node_modules/@clr/icons/clr-icons.min.js"
+            ]
+          },
+          "configurations": {
```

Comment:
Please verify you can still run all the compilation variants:

```
ui$ ./project.rb build --env local
ui$ ./project.rb build --env test
...
ui$ ./project.rb build --env prod
```

See comment here about the expected behavior, i.e. we expect optimizations in staging,stable,prod:
https://github.com/all-of-us/workbench/blob/47bfb716834517cfd415153c7429be0d66ebe10b/common-ui/common-ui-devstart.rb#L46

Reply:
Testing now...

Reply:
Okay, this all works!